oneview_redfish_toolkit/blueprints/manager.py

```diff
@@ -91,3 +82,26 @@ def get_managers(uuid):
         # In case of error log exception and abort
         logging.exception('Unexpected error: {}'.format(e))
         abort(status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+def _get_appliance_by_uuid(uuid, oneview_appliances):
+    appliance = dict()
+    appliance_index = 0
+
+    for index, ov_appliance in enumerate(oneview_appliances):
+        if ov_appliance["uuid"] == uuid:
+            appliance = ov_appliance
+            appliance_index = index
+            break
+
+    if not appliance:
+        raise OneViewRedfishError("Could not find manager with id " + uuid)
+
+    return appliance, appliance_index
+
+
+def get_current_manager():
```

Comment:
My suggestion is put this helper methods inside a service file (in services folder) to avoid referencing this blueprint in another blueprint files.

For example this method `get_current_manager` is called in others blueprint files.

With this we are organizing the helper codes and services in a specific files and folder to do that.